src/components/message/message.spec.tsx

```diff
@@ -201,4 +202,80 @@ describe("Message", () => {
   describe("styled-system", () => {
     testStyledSystemMargin((props) => <Message {...props} />);
   });
+
+  describe("focus behaviour", () => {
+    let wrapper: ReactWrapper | ShallowWrapper;
+    let onDismissCallback: jest.Mock;
+
+    beforeEach(() => {
+      const element = document.createElement("div");
+      const htmlElement = document.body.appendChild(element);
```

Comment:
do we need the `appendChild` stuff here? Fair enough if we do, I know we use it elsewhere but I've never fully understood why it's sometimes needed, and I'd rather do without if we can. If it doesn't work without, feel free to leave it in.